lib/main.php

```diff
@@ -186,6 +186,10 @@ function convert ($body, $query_string, $set_properties, $is_skip_variations = f
           $entry['link'] .= ($query_string ? '&' : '?');
           $entry['link'] .= 'variation_id=' . $body['_id'];
         }
+        if ($utm) {
+          $entry['link'] .= ($query_string ? '&' : '?');
+          $entry['link'] .= 'campaign_utm=' . $utm;
```

Comment:
```suggestion
          $entry['link'] .= 'utm_campaign=' . $utm;
```

Tem alguma razão pra não usar o convencionado `utm_campaign`?